.github/validation_samples/kaon_enhanced/config.py

```diff
@@ -0,0 +1,162 @@
+import os
+import sys
+
+from LDMX.Framework import ldmxcfg
+
+thisPassName = 'test'
+p=ldmxcfg.Process(thisPassName)
+
+p.termLogLevel = 0
+# This need to be set but has no meaning given p.totalEvents
+p.maxEvents = 1
+p.totalEvents = int(0.5*int(os.environ['LDMX_NUM_EVENTS']))
```

Comment:
It takes about double time to have the kaons done, i.e. with taking 1/2 of the events it will take about the same time for validation

Reply:
done in next push